test/uk/gov/hmrc/merchandiseinbaggage/controllers/PreviousDeclarationDetailsControllerSpec.scala

```diff
@@ -16,32 +16,30 @@
 
 package uk.gov.hmrc.merchandiseinbaggage.controllers
 
+import com.google.inject.Inject
 import org.mockito.ArgumentMatchersSugar.any
 import org.mockito.MockitoSugar.{mock, when}
 import play.api.test.Helpers._
 import uk.gov.hmrc.http.HeaderCarrier
 import uk.gov.hmrc.merchandiseinbaggage.CoreTestData
-import uk.gov.hmrc.merchandiseinbaggage.config.MibConfiguration
 import uk.gov.hmrc.merchandiseinbaggage.connectors.MibConnector
 import uk.gov.hmrc.merchandiseinbaggage.controllers.routes._
 import uk.gov.hmrc.merchandiseinbaggage.model.api.{Declaration, DeclarationType, Paid, SessionId}
 import uk.gov.hmrc.merchandiseinbaggage.model.core.DeclarationJourney
 import uk.gov.hmrc.merchandiseinbaggage.service.MibService
-import uk.gov.hmrc.merchandiseinbaggage.stubs.MibBackendStub.givenPersistedDeclarationIsFound
+import uk.gov.hmrc.merchandiseinbaggage.stubs.MibBackendStub
 import uk.gov.hmrc.merchandiseinbaggage.views.html.PreviousDeclarationDetailsView
 
 import scala.concurrent.ExecutionContext.Implicits.global
 import scala.concurrent.Future
 
-class PreviousDeclarationDetailsControllerSpec
-    extends DeclarationJourneyControllerSpec
-    with CoreTestData
-    with MibConfiguration {
+class PreviousDeclarationDetailsControllerSpec extends DeclarationJourneyControllerSpec with CoreTestData {
 
   val mockNavigator: Navigator   = mock[Navigator]
   val mockMibService: MibService = mock[MibService]
 
   val view: PreviousDeclarationDetailsView = app.injector.instanceOf[PreviousDeclarationDetailsView]
+  private val stub                         = app.injector.instanceOf[MibBackendStub]
```

Comment:
Once the changes in the `MibBackendStub` have been done, please revert these stub changes, thanks